File: Beecrowd/1021/1021.c
#include <stdio.h>

int main(){
    int notes[] = {10000, 5000, 2000, 1000, 500, 200};
    int coins[] = {100, 50, 25, 10, 5, 1};
    int reais, cents;

    scanf("%d.%d", &reais, &cents);
    reais = 100 * reais + cents;

    printf("NOTAS:\n");
    for(int i = 0; i < 6; ++i){
        printf("%d nota(s) de R$ %.2lf\n", reais/notes[i], notes[i]/100.0);
        reais %= notes[i];
    }

    printf("MOEDAS:\n");
    for(int i = 0; i < 6; ++i){
        printf("%d moeda(s) de R$ %.2lf\n", reais/coins[i], coins[i]/100.0);
        reais %= coins[i];
    }

    return 0;
}

File: Beecrowd/1021/1021.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1021
money = float(input())
money *= 100

notes = [10000, 5000, 2000, 1000, 500, 200]
coins = [100, 50, 25, 10, 5, 1]

print('NOTAS:')
for note in notes:
    print(f'{int(money//note)} nota(s) de R$ {(note/100):.2f}')
    money %= note

print('MOEDAS:')
for coin in coins:
    print(f'{int(money//coin)} moeda(s) de R$ {(coin/100):.2f}')
    money %= coin

